### ISSUE TITLE
Improvement for k8s.io/docs/tasks/tools/install-kubectl-macos/

### ISSUE THREAD

**Issue by logsnaath** [kubernetes/website#27579]
```
echo 'alias k=kubectl' >>~/.bash_profile
echo 'complete -F __start_kubectl k' >>~/.bash_profile

```
The alias "k" command completion works only after a command completion is tried with kubectl first.

```
$ kc -bash: completion: function `__start_kubectl' not found
$ kubectl
alpha          apply          certificate    convert        delete         edit  
...
$ kc
alpha          apply          certificate    convert        delete         edit 
...
```

**Comment by tengqm** [comment]
This sounds like out of scope of the documentation. A user may give `kubectl` arbitrary alias.
/close
